code/mission/missionparse.cpp

```diff
@@ -5447,6 +5463,10 @@ void parse_one_background(background_t *background)
 		stuff_float(&sle.ang.b);
 		stuff_float(&sle.ang.h);
 
+		// correct legacy bitmap angles which used incorrect math in older versions
+		if (!background->flags[Starfield::Background_Flags::Corrected_angles_in_mission_file])
+			stars_correct_background_angles(&sle.ang);
```

Comment:
I wonder if it might be worth writing to the debug log whether we're making this correction. Ditto for the identical code below while parsing data on background bitmaps.

Reply:
I don't think that's necessary, but I'll leave the PR open in case anyone else wants to comment.